### PR TITLE
[GLOSSARY] change

### PR DESCRIPTION
**Pull request by MaxHillebrand**
Adds glossary of change coins

### CODE REVIEW

File: docs/glossary/Glossary.md [hunk 1]
```diff
@@ -20,6 +20,14 @@ A design document for introducing features or information to Bitcoin.
 Read more: [Wasabi Supported BIPs](/using-wasabi/BIPs.md)
 :::
 
+:::details
+### Change
+
+When in a spending transaction the provided input coins are larger than the value send to the destination address, then the leftover change is send back to an address of the same wallet.
```
Comment (SimoneBovi) [inline docs/glossary/Glossary.md]: When in a spending transaction the provided input coins are larger than the value sent to the destination address, then the leftover change is sent back to an address of the same wallet.

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
When in a spending transaction the provided input coins are larger than the value sent to the destination address, then the leftover change is sent back to an address of the same wallet.
```

**Comment by RiccardoMasutti** [comment]
NACK.
I think Bitcoin basics should not be in Wasabi glossary.

File: docs/glossary/Glossary.md [hunk 2]
```diff
@@ -20,6 +20,14 @@ A design document for introducing features or information to Bitcoin.
 Read more: [Wasabi Supported BIPs](/using-wasabi/BIPs.md)
 :::
 
+:::details
+### Change
+
+When in a spending transaction the provided input coins are larger than the value sent to the destination address, then the leftover change is sent back to an address of the same wallet.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
In a spending transaction where the provided input coins are larger than the value sent to the destination address, then the leftover change is sent to a new address of the same wallet.
```

File: docs/glossary/Glossary.md [hunk 3]
```diff
@@ -20,6 +20,14 @@ A design document for introducing features or information to Bitcoin.
...
+CoinJoins create standardized equal value outputs, and the change of a CoinJoin can easily be linked to the input, thus they have 0 anonymity set.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]: I think this sentence should be removed from this definition.

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]: I was thinking the same... 
```suggestion
```

**Comment by MaxHillebrand** [comment]
`Change` is mentioned many times in the docs, it even has it's own chapter because of the many privacy problems with it, so I think definitely it should be included.

**Pull request merged**